### ISSUE TITLE
The ⋮ Menu is Hard to see on Account Management Screen

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#9725]
## Description 
From the _Account Management Screen_ it is difficult to find the ⋮ to manage particular entries.

![account-management](https://user-images.githubusercontent.com/815158/81589309-f394d080-937e-11ea-9dcd-174ebfcbe086.gif)

## Steps to Reproduce
1. Visit the _Account Management Screen_
2. Hover an account entry

## Actual result:
The ⋮ menu is difficult to see.

## Expected result:
It should be clear to the user how to manage account entries. One proposal would be to carry the ⚙ icon from the profile-dropdown into the UI:

![image](https://user-images.githubusercontent.com/815158/81590322-458a2600-9380-11ea-935a-2ee6947c30bf.png)

![image](https://user-images.githubusercontent.com/815158/81590003-c3016680-937f-11ea-8f53-223692676e3e.png)

## Brave version (brave://version info)
1.8.95

**Comment by rebron** [comment]
Closing as stale.
